packages/trpc/server/routers/loggedInViewer/connectedCalendars.handler.ts

```diff
@@ -48,6 +54,19 @@ export const connectedCalendarsHandler = async ({ ctx }: ConnectedCalendarsOptio
       So create a default destination calendar with the first primary connected calendar
       */
     const { integration = "", externalId = "", credentialId } = connectedCalendars[0].primary ?? {};
+    // Select the first calendar matching the primary by default since that will also be the destination calendar
```

Comment:
@CarinaWolli We can't do that in the client side because the `isSelected` value comes from the db, so i have added a new optional query parameter to just do this in the onboarding screen. There is one more bug https://github.com/calcom/cal.com/blob/main/apps/web/pages/settings/my-account/calendars.tsx#L190 because of which the first item is always enabled in `apps/installed/calendar`.